### DISCUSSION TITLE
<Table> component scrollToRow doesn't scroll to specific row on table initiation

### DISCUSSION THREAD

**Discussion by DenisMicha** [iTwin/iTwinUI#1885] [Q&A]
I am using a `<Table>` component and its **scrollToRow** property to scroll to a specific row immediately when my table appears. However, I notice that it doesn't scroll on table appearance (initiation). Is the behaviour intended or is it a bug?

**Comment by r100-stack** [comment]
I am not sure I understood your question. Can you elaborate? Would also be helpful if you can show what you're asking using a short screen-recording and/or a repro by forking [this sandbox](https://stackblitz.io/github/iTwin/iTwinUI/tree/main/minimal-sandbox?file=/src/App.tsx).

Also, you could visit our [Table → ScrollToRow story](https://itwin.github.io/iTwinUI/react/?story=table--scroll-to-row) for more information about this feature.

**Reply by DenisMicha** [reply]
It seems I just needed to update to newest itwinui-react version and it solved my issue, so this is not relevant anymore.
